text/logs/0130-logs-1.0ga-definition.md

```diff
@@ -0,0 +1,57 @@
+# Logs 1.0 GA Scope
+
+This document defines what's in scope for OpenTelemetry Logs 1.0 GA.
+
+## Motivation
+
+Clearly defined scope is important to align all logs contributors and to make
+sure we know what target we are working towards. Note that some of the listed
+items are already fully or partially implemented but are still listed for
+completeness.
+
+## Logs 1.0 GA Todo List
+
+### Specification and SDK
+
+- Write guidelines and specification for logging libraries to support
+  OpenTelemetry-compliant logs.
+
+- Implement OpenTelemetry logs SDK for Java that support trace context
+  extraction from the current execution context.
+
+- Implement appenders for Log4J using OpenTelemetry SDK that automatically
```

Comment:
Why Log4J and not SLF4J

Reply:
appenders are not part of the slf4j api, they are implementor specific (e.g. ch.qos.logback.core.Appender or org.apache.logging.log4j.core.Appender)

Reply:
I have no particular preference for which Java logging library to support. I can change to SFL4J if it is a more popular/preferable library for the community.
The goal here is not to have comprehensive coverage of all Java application needs, but rather to set an example and clearly demonstrate how OpenTelemetry logging SDK can be used with a logging library.

Reply:
I would say appenders for at least one of the major logging library?

Reply:
That's a good  point, let me rephrase to avoid mentioning a specific library. The decision on which library to support and how exactly does not need to happen right now.

Reply:
Re-worded, please check again.

Reply:
I think that it's important to have a solution that works generally- in the Java world, at least Log4j, Log4j2, Logback, and java.util.logging (though obviously we start with one).

Reply:
Sounds good, I think the text now reflects this.